Make.config

```diff
@@ -333,9 +333,9 @@ JENKINS_RESULTS_DIRECTORY ?= $(abspath $(TOP)/jenkins-results)
 # Clone files instead of copying them on APFS file systems. Much faster.
 CP:=$(shell df -t apfs / >/dev/null 2>&1 && echo "cp -c" || echo "cp")
 
-MONO_IOS_FILENAME:=ios-release-Darwin-$(MONO_HASH).zip
+MONO_IOS_FILENAME:=ios-release-Darwin-$(MONO_HASH).7z
```

Comment:
Just out of curiosity: why `7z` instead of `zip`? Are the archives smaller (how much?)?

Reply:
@rolfbjarne yes they are _much_ smaller. The ios archive went from 688MB to 272MB.